eldritch/items/__init__.py

```diff
@@ -0,0 +1,70 @@
+from eldritch.assets import Deputy
+from .base import Item, Weapon, OneshotWeapon, Tome
+from .common import (
+    AncientTome, Automatic45, Axe, DarkCloak,
+    Derringer18, Revolver38, Dynamite,
+    TommyGun, Food, ResearchMaterials,
+    Rifle, Shotgun, CavalrySaber, Whiskey, Knife, Lantern,
+    Bullwhip, Cross)
+
+from .unique import (
+    HolyWater, EnchantedBlade, EnchantedKnife, MagicLamp, MagicPowder, SwordOfGlory
+)
+from .spells import (BindMonster, DreadCurse, EnchantWeapon,
+                     FindGate, FleshWard, Heal, Mists, RedSign,
+                     Shrivelling, Voice, Wither)
+from .deputy import PatrolWagon, DeputysRevolver
+
+
+def CreateCommon():
```

Comment:
Similarly, I recommend moving this to `common.py` - makes it so that if you add a new item, there's just fewer files to update.